studio-docs/source/sharing-graphs.mdx

```diff
@@ -0,0 +1,569 @@
+
+<img class="screenshot" alt="GraphQL codeblock in README" src="./img/readme-codeblock.jpg" width="300" />
+
+
+## Public graphs
+
+You can make individual [variants](./org/graphs/#managing-variants) of your graph public. If you do, **anyone with that variant's public link** can view the following pages for that variant in Studio:
+
+* Home
+* Schema
+* Explorer
+
+This enables consumers of your graph to learn about your graph's schema and any special usage information (such as authentication details) that you've documented in [the variant's README](#the-readme-page). They can also run properly authenticated operations against your graph with the Explorer.
+
+People outside your organization can't view any _other_ pages for a public variant (Fields, Operations, etc.), and they can never view _any_ pages for private variants. New variants always start as private.
+
+### Making a variant public
+
+> **Note:** Only organization members with the [`Org Admin` or `Graph Admin` role](./org/members/) can toggle a variant's visibility.
+
+1. Go to your graph's Settings page and open the Access tab.
+2. Under the Variants section, toggle the switch for the variant you want to make public.
+3. Confirm your selection.
+
+You can toggle the switch back to make the variant private again.
+
+## Embedding the Explorer
+
+If you have a [public variant](#public-graphs) of your graph, you can embed the [Apollo Studio Explorer](./explorer/) in a webpage that you can then provide to your graph's consumers. This enables those consumers to test out operations from your own website.
+
+For example, here's an embedded Explorer for an Apollo example graph:
+
+<EmbeddableExplorer
+  graphRef="Apollo-Fullstack-Demo-o3tsz8@current"
+  graphEndpoint="https://apollo-fullstack-tutorial.herokuapp.com/"
+  graphSubscriptionEndpoint="wss://apollo-fullstack-tutorial.herokuapp.com/graphql"
+  styles={{width: '100%', height:'450px', border: '0px', borderRadius: '4px'}}
+/>
+
+> This embedded Explorer collapses its left column by default because the full three-column layout is a little cramped on this page. You can [customize the embedded Explorer's appearance](#options) to suit the page it's on.
+
+### Basics
+
+You embed the Explorer in an `iframe` that points to `https://explorer.embed.apollographql.com`. You add a `graphRef` query parameter to this URL to indicate which [public variant](#public-graphs) the Explorer should use:
+
+```jsx
+<iframe src="https://explorer.embed.apollographql.com/?graphRef=my-graph@my-variant" />
+```
+
+The embedded Explorer uses this `graphRef` to fetch your public variant's schema and its GraphQL endpoint URL from Apollo Studio.
+
+#### Handling CORS
+
+By default, the embedded Explorer sends user-executed operations directly to your variant's GraphQL endpoint. Because the Explorer is embedded in an `iframe`, the origin of those network requests is `https://explorer.embed.apollographql.com`.
```

Comment:
```suggestion
The embedded Explorer can either send user-executed operations directly to your variant's GraphQL endpoint, or send them to your webpage via the `postMessage` API. If you pass `postMessageOperations=true`, because the Explorer is embedded in an `iframe`, the origin of those network requests is `https://explorer.embed.apollographql.com`.
```